src/main/java/com/softserve/academy/spaced/repetition/domain/LearningRegime.java

```diff
@@ -0,0 +1,6 @@
+package com.softserve.academy.spaced.repetition.domain;
+
+public enum LearningRegime {
+    STATUS_LEARNING,
```

Comment:
No need for suffix _LEARNING as it is already present in enum name. Check the difference:

- LearningRegime.STATUS_LEARNING

- LearningRegime.STATUS